addon/components/power-select-multiple/trigger.js

```diff
@@ -70,7 +70,10 @@ export default Component.extend({
   }),
 
   maybePlaceholder: computed('placeholder', 'select.selected.length', function() {
-    if (isIE) {
+    let component = this.get('placeholderComponent');
+    let hasCustomPlaceholder = component !== 'power-select/placeholder';
```

Comment:
Since you are not going to render this placeholder if the search is disabled, I think that you don't need this code.